app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is used to store references to `Person` objects by their names. Ensure that names are unique to avoid overwriting existing entries.